packages/api/cms-api/src/access-log/access-log.module.ts

```diff
@@ -23,14 +24,18 @@ interface AccessLogModuleOptions {
     ],
 })
 export class AccessLogModule {
-    static forRoot({ shouldLogRequest }: AccessLogModuleOptions): DynamicModule {
```

Comment:
I decided against a more general `disableLogging` or `enableLogging` option to prevent mistakes